src/lib/terra/lcd/classic/index.ts

```diff
@@ -0,0 +1,47 @@
+import { AxiosRequestConfig } from 'axios'
+import { Lcd, PoolInfo, TokenInfo } from '../interfaces';
+import { ClassicCosmos46Lcd } from './classic.cosmos46';
+import { ClassicLegacyLcd } from './classic.legacy';
+
+export class ClassicLcd implements Lcd {
+    private clients: Lcd[];
+    private indicator = 0;
+    constructor(url?: string, config?: AxiosRequestConfig) {
+        this.clients = [
+            new ClassicCosmos46Lcd(url, config),
```

Comment:
Nit: `cosmos46` and `legacy` are a bit confusing to figure out their order. how about just remove "legacy" and just let it be ClassicLcd if it is the latest classic one?